projects/storefrontlib/cms-components/myaccount/address-book/address-book.module.ts

```diff
@@ -37,6 +38,12 @@ import { defaultSuggestedAddressesDialogLayoutConfig } from './address-form/sugg
         AccountAddressBookComponent: {
           component: AddressBookComponent,
           guards: [AuthGuard],
+          providers: [
+            {
+              provide: AddressBookComponentService,
+              useClass: AddressBookComponentService,
+            },
+          ],
```

Comment:
Is it _really_ needed?

Reply:
this might get replaced with the UserAddressEffect override